src/intl/primary-language-subtag.js

```diff
@@ -0,0 +1,16 @@
+import { iso6391To6393, iso6393 } from './iso639.js'
+
+/**
+ * Normalize a primary language subtag to ISO 639-3 if possible.
+ * @param {string} subtag - The primary language subtag to normalize.
+ * @returns {string | null} - The normalized ISO 639-3 language code, or null if the input is not valid.
+ */
+export function normalizePrimaryLanguageSubtag(subtag) {
```

Comment:
Do all these utils need to be in separate files? Feels like they could be wrapped inside `intl/index.js`